app/styles/app.scss

```diff
@@ -291,3 +291,47 @@ strong {
     color: inherit;
   }
 }
+
+.activity-customer-visible-icon {
+  position: relative;
+  float: right;
+  margin-top: -22px;
+  margin-left: 92%;
+  z-index: 1;
+}
+
+.customer-visible-icon {
+  position: relative;
+  float: right;
+  margin-top: -27px;
+  margin-right: 10px;
+  z-index: 1;
+}
+
+@media (max-width: 480px) {
+  .activity-customer-visible-icon {
+    margin-top: 8px;
+    margin-left: 85%;
+  }
+}
+
+@media #{$xs-viewport} {
+  .activity-customer-visible-icon {
+    margin-top: -10px;
+    margin-left: 88%;
+  }
+}
+
+@media #{$sm-viewport} {
+  .activity-customer-visible-icon {
+    margin-top: -30px;
+    margin-left: 90%;
+  }
+}
+
+@media (max-width: 680px) {
+  .activity-customer-visible-icon {
+    margin-top: 0px;
+    margin-left: 85%;
+  }
+}
```

Comment:
Those margins seem oddly specific, have you tried using `em` or `rem`?

Reply:
No, I didn't. Changed it now.

Reply:
No idea how it looks, I didn't check it out :wink: However, CSS should be written a) mobile first and b) not with this many media queries for something as simple as margin. You can leave it, I don't really mind. Just try keeping it simple in the future.